### ISSUE TITLE
Connect with RoC only

### ISSUE THREAD

**Issue by JFGHT** [Josko/aura-bot#67]
Is it possible with the current version to launch the bot RoC only mode?

Edit: Ok, it's not. I'll try to do it myself.

**Comment by Josko** [comment]
Yeah,  there was the #46 but I'm curious what do people need RoC for anyway? Is it _that_ popular?

**Comment by JFGHT** [comment]
Yes it definitely is. Just log in into battle.net and type /users (or check rocarena.com & w3replayers.com).

I opened a pull request with the changes but I'm not sure it's working well.
